src/controllers/queries/index.ts

```diff
@@ -8,6 +8,33 @@ import QueriesTemplates from '../../views/queries.js'
 import QueryListTemplates from '../../views/queriesList.js'
 import { HTML, HTMLController } from '../HTMLController.js'
 
+type QueryStatus = 'resolved' | 'pending_your_input' | 'pending_their_input'
+interface Query {
+  company_name: string
+  query_type: string
+  updated_at: Date
+  status: QueryStatus
+}
+type QuerySubset = {
+  updated_at: Date
+  connection_id: string
+  status: 'resolved' | 'pending_your_input' | 'pending_their_input'
+  id: string
+  created_at: Date
+  query_type: string
+}
+
+type Connection = {
```

Comment:
Does this type not exist somewhere else? Can you just import it?

Reply:
not that I could see ..the closest there is is the connection interface in views>connection.tsx but that one is missing id `interface connection {
  company_name: string
  status: ConnectionStatus
}`

Reply:
maybe
```ts
import type { ConnectionRow } from '../../models/db/types.js`
```
Likely you can export the Query type there as well maybe so it's located in a more shared place assoicated with where it comes from?

Reply:
thank you! I'll change it